profile/src/main/java/com/scottlogic/deg/profile/dto/FieldDTO.java

```diff
@@ -18,10 +18,14 @@
 
 public class FieldDTO {
     public String name;
+    public String formatting;
+    public Boolean unique;
 
     public FieldDTO() {}
```

Comment:
both of these constructors can be removed, and the tests all still pass
if that's done, can also change from Boolean to boolean, and remove the null check